sdcm/sct_config.py

```diff
@@ -309,13 +309,20 @@ class SCTConfiguration(dict):
         dict(name="nemesis_during_prepare", env="SCT_NEMESIS_DURING_PREPARE", type=str,
              help="""Run nemesis during prepare stage of the test"""),
 
+        dict(name="nemesis_run", env="SCT_NEMESIS_RUN", type=str,
```

Comment:
What is the motivation behind adding this new parameter? Why not to use `NoOpMonkey`?

Reply:
Add to class that start nemesis thread parameter "start" = True/False.
In scale test in case NoOpMonkey - don't start nemesis thread

Reply:
We discussed today that you will remove this and use noopmonkey check

Reply:
done